### PR TITLE
docs(inputs): Add plugin metadata and update description for a* to f*

### PR DESCRIPTION
**Pull request by srebhan**
## Summary

This PR adds some metadata to the input plugins starting with `a` to (including) `f`  like the version this plugin was introduced, labels and the supported operating systems. Furthermore, the PR unifies the plugin descriptions a bit and uses [Github Markdown alerts](https://docs.github.com/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts) for notes and other important information.

> [!NOTE]
> This is part of a bigger effort to automatically generate (parts) of our [documentation website](https://docs.influxdata.com/telegraf) for making the GitHub page the primary source of thruth.

## Checklist

- [x] No AI generated code was used in this PR

### CODE REVIEW

**Comment by srebhan** [comment]
@DStrand1 please ignore the `lint plugin readmes` error as this is due to the readme_linter checking source code line-length...

File: plugins/inputs/conntrack/README.md [hunk 1]
```diff
@@ -1,40 +1,25 @@
 # Conntrack Input Plugin
```
Comment (DStrand1) [inline plugins/inputs/conntrack/README.md:1]:
```suggestion
# Netfilter Conntrack Input Plugin
```

File: plugins/inputs/conntrack/README.md [hunk 2]
```diff
@@ -1,40 +1,25 @@
...
+of directory and filenames can be specified. Any locations that doesn't exist
+is ignored.
+
+⭐ Telegraf v1.0.0
+🏷️ system
+💻 linux
 
-For more information on conntrack-tools, see the
-[Netfilter Documentation](http://conntrack-tools.netfilter.org/).
+[conntrack]: http://conntrack-tools.netfilter.org/
```
Comment (DStrand1) [inline plugins/inputs/conntrack/README.md:22]:
```suggestion
[conntrack]: https://conntrack-tools.netfilter.org/
```

File: plugins/inputs/ethtool/README.md [hunk 3]
```diff
@@ -1,7 +1,11 @@
 # Ethtool Input Plugin
 
-The ethtool input plugin pulls ethernet device stats. Fields pulled will depend
-on the network device and driver.
+This plugin collects ethernet device statistics. The available information
+strongly depend on the network device and driver.
```
Comment (DStrand1) [inline plugins/inputs/ethtool/README.md:4]:
```suggestion
strongly depends on the network device and driver.
```

File: plugins/inputs/eventhub_consumer/README.md [hunk 4]
```diff
@@ -1,6 +1,14 @@
-# Event Hub Consumer Input Plugin
+# Azure Event Hub Consumer Input Plugin
 
-This plugin provides a consumer for use with Azure Event Hubs and Azure IoT Hub.
+This plugin allows to consumer messages from [Azure Event Hubs][eventhub] and
```
Comment (DStrand1) [inline plugins/inputs/eventhub_consumer/README.md:3]:
```suggestion
This plugin allows consuming messages from [Azure Event Hubs][eventhub] and
```

File: plugins/inputs/fluentd/README.md [hunk 5]
```diff
@@ -1,27 +1,23 @@
...
-  @type http
-  @id http
-  port 8888
-</source>
-```
-
-[1]: https://docs.fluentd.org/input/monitor_agent
-[2]: https://docs.fluentd.org/configuration/config-file#common-plugin-parameter
+This plugin gathers internal metrics of a [fluentd][fluentd] instance provided
+by fluentd's [monitor agent plugin][monitor_agent]. It understands data provided
```
Comment (DStrand1) [inline plugins/inputs/fluentd/README.md:4]:
```suggestion
by fluentd's [monitor agent plugin][monitor_agent]. Data provided
```

**Review by DStrand1 — APPROVED** [review]

**Pull request merged**